front/report.infocom.conso.php

```diff
@@ -54,7 +54,8 @@
 $stat = new Stat();
 $chart_opts =  [
    'width'  => '90%',
-   'legend' => false
+   'legend' => false,
+   'title'  => ['Year','Value']
```

Comment:
Not sure a title is mandatory here.

Reply:
this was a way to have good first line on CVS export, I might use 'headers' instead of 'title'